mrp_product_document_link/models/mrp_production.py

```diff
@@ -0,0 +1,31 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    document_urls = fields.Text(
```

Comment:
Use a more specific name for the field (and its label as well):
```suggestion
    product_document_urls = fields.Text(
```